migrate/labels.yml

```diff
@@ -0,0 +1,122 @@
+#   label would be useful to have in the entire organization, we can always add
+#   it here post facto.
+# * We have added these based on need. They are not all-encompassing. If something
+#   seems missing, feel free to open a PR.
+# * Whenever a repo has a matching label, that label will be updated. If there
+#   is no matching label, the label will be created anew. "Matching" is determined
+#   by comparing the label's lowercased name, with all puncutation, emoji, and special
+#   chars removed.
+# * For picking hex colors, you might use: https://www.htmlcolor-picker.com/
+#   Describe your color with a comment so it's easier to review.
+
+
+### LABELS INDICATING HOW ISSUES CAN BE ENAGAGED WITH.
+
+- name: "good first issue"
+  color: "42dd35"  # friendly lime green
+  description: "A good task for a newcomer to start with"
+
+- name: "help wanted"
+  color: "54976d"  # fenway green
+  description: "Ready to be picked up by anyone in the community"
+
+
+### LABELS INDICATING BROAD THEMES OF WORK.
+### MORE THAN ONE OF THESE MAY APPLY AT A TIME.
+
+- name: "maintenance"
+  color: "ff9125"  # construction orange
+  description: "Relates to platform health and/or routine upkeep"
+
+- name: "security"
+  color: "f0d165"  # caution yellow
+  description: "Relates to improving to the security posture of the platform"
+
+- name: "enhancement"
+  color: "a2eeef"  # electric blue!
+  description: "Relates to new features or improvements to existing features"
+
+- name: "documentation"
+  color: "0052cc"  # royal blue
+  description: "Relates to documentation improvements"
+
+
+### LABELS INDICATING THE SCOPE OR FUNCTION OF THE ISSUE.
+### AT MOST ONE OF THESE SHOULD BE USED AT A TIME.
+
+- name: "depr"
+  color: "3c3cc3"  # indigo-blue
+  description: "Proposal for deprecation & removal per OEP-21"
+
+- name: "bug"
+  color: "d93f0b"  # scarlet red...
+  description: "Report of or fix for something that isn't working as intended"
+
+- name: "discovery"
+  color: "d876e3"  # a curious shade of lavendar
+  description: "Pre-work to determine if an idea is feasible"
+
+- name: "epic"
+  color: "7e027e"  # epic violet
+  description: "Large unit of work, consisting of multiple tasks"
+
+- name: "initiative"
+  color: "340963"  # formidably deep mauve
+  description: "Huge unit of work, consisting of multiple epics"
+
+
+### LABELS INDICATING ISSUE OUTCOMES.
+
+- name: "wontfix"
+  color: "ffffff"  # white
+  description: "This will not be worked on"
+
+- name: "duplicate"
+  color: "cfd3d7"  # duplicitous grey
+  description: "This issue or pull request already exists elsewhere"
+
+
+### LABELS USED IN THE OPEN EDX CONTRIBUTION WORKFLOW.
+### PLEASE CONSULT WITH OPEN EDX PROJECT MANAGMENT BEFORE CHANGING THESE.
+
+- name: "waiting on author"
+  color: "bfd6f6"  # baby blue
+  description: "PR reviewers are waiting on the author to answer questions, fix tests, etc."
+
+- name: "inactive"
```

Comment:
Could we use "author inactive" or "author unresponsive" here? I feel like "inactive" is a little ambiguous as to what/who is inactive.

Reply:
As with all labels, I'm wondering who is adding this label, and who is looking for it?  Whoever is marking a PR as "inactive", why wouldn't they close the PR?  Is the label meant to be bookkeeping on closed PRs?  Would we ever have an open PR with the "inactive" label?

Reply:
... and I just saw "closed inactivity" below.

Reply:
Each label in the `LABELS USED IN THE OPEN EDX CONTRIBUTION WORKFLOW` section already exists in any repo that the OSPR bot has interacted with (which is most repos). So, these labels are not adding anything new, they are just pulling what the OSPR bot has already been doing in labels.yml.

> Whoever is marking a PR as "inactive", why wouldn't they close the PR? Is the label meant to be bookkeeping on closed PRs? Would we ever have an open PR with the "inactive" label?

@mphilbrick211 and @itsjeyd, who use all the labels in this section, explained how the labels are used [on the italicized notes on this table](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3695214629/openedx+org-wide+labels#2.-Proposal%3A-List-of-starter-labels).  I think that I will copy those notes into the YAML file as comments.

[1] "most" being any repo that the OSPR bot has interacted with, so any repo with non-2U/Axim contributions.